src/Data/Fin/Permutation.agda

```diff
@@ -172,6 +196,17 @@ module _ {m n} (π : Permutation (suc m) (suc n)) where
     punchIn (πʳ (πˡ i)) (remove (πˡ i) π ⟨$⟩ʳ j)  ≡⟨ cong₂ punchIn (inverseʳ π) refl ⟩
     punchIn i (remove (πˡ i) π ⟨$⟩ʳ j)            ∎
 
+  lift-remove : zero ≡ πʳ zero → ∀ i → lift₀ (remove zero π) ⟨$⟩ʳ i ≡ πʳ i
```

Comment:
The code would probably read easier if you replaced `zero` everywhere with https://github.com/agda/agda-stdlib/blob/f831cb2bda71f0e06f2b664d006a981be8bdc853/src/Data/Fin/Patterns.agda#L16